binding.gyp

```diff
@@ -5,7 +5,8 @@
       "sources": [ 
         "bindings/profiler.cc",
       ],
-      "include_dirs": [ "<!(node -e \"require('nan')\")" ]
+      "include_dirs": [ "<!(node -e \"require('nan')\")" ],
+      "cflags": [ "-Wno-cast-function-type" ]
```

Comment:
It would be good to add a comment here on why it's added.

Reply:
Thanks!
https://github.com/googleapis/cloud-profiler-nodejs/pull/428